core/os/device/device.proto

```diff
@@ -204,12 +204,34 @@ message OpenGLDriver {
 
 // VulkanDriver describes the device driver support for the Vulkan API.
 message VulkanDriver {
-    // Supported extensions.
-    repeated string Extensions = 1;
-    // Driver name. e.g. "Adreno (TM) 320".
-    string Renderer = 2;
-    // Driver vendor name. e.g. "Qualcomm".
-    string Vendor = 3;
-    // Renderer version. e.g. "OpenGL ES 3.0 V@53.0 AU@  (CL@)".
-    string Version = 4;
+    // Enumerated instance layers.
+    repeated VulkanLayer Layers = 1;
+    // Instance extensions provided by Vulkan implementations and implicit
+    // layers.
+    repeated string IcdAndImplicitLayerExtensions = 2;
```

Comment:
What does Icd mean / stand for?

Reply:
It stands for Vulkan Installable Client Driver, I think this is just the implementation of Vulkan (drivers). Icd seems to be used quite widely in Vulkan docs.